README.md

```diff
@@ -1,218 +1,197 @@
+![Continuous Integration](https://github.com/tournesol-app/tournesol/workflows/Continuous%20Integration/badge.svg?branch=master)
 
-See the wiki page [Contribute to Tournesol](https://wiki.tournesol.app/index.php/Contribute_to_Tournesol) for details.
+This repository hosts the code of the platform 🌻[Tournesol.app](https://tournesol.app). See the wiki page [Contribute to Tournesol](https://wiki.tournesol.app/index.php/Contribute_to_Tournesol) for details.
 
 ![Home page of Tournesol.app](https://user-images.githubusercontent.com/10453308/115123905-9b6b4300-9fbf-11eb-8853-25552d13f7b0.png)
 
 We use [TensorFlow](http://tensorflow.org/) to compute the aggregated scores,
 [Django](https://www.djangoproject.com/) for the backend, and [React.js](https://reactjs.org/) for the front-end.
 
-## How to launch (tested on Ubuntu in WSL and on Ubuntu 20.04)
+![image](https://user-images.githubusercontent.com/1012270/119049451-13a69900-b9b0-11eb-807f-25f2455f3d05.png)
 
-![Continuous Integration](https://github.com/tournesol-app/tournesol/workflows/Continuous%20Integration/badge.svg?branch=master)
+## Development workflow
+To contribute to the code base, you will need basic knowledge of the [Linux command line](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview), [Git](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners) (specifically, branches) and GitHub [Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+
+❓ If unsure what to do, ask on [Discord](https://discord.gg/ajJYCsnQ).
+
+- All development is done locally on developers' machines
+- The `master` branch contains code that can be run without any tweaks or modifications. This branch is protected against direct commits. Each change needs to be done in a separate branch, which is then used in a Pull Request (PR). The PR needs to pass automated tests (Github Continuous Integration/CI), as well as a review from one of the members of the 🇨🇭[Tournesol Association](https://wiki.tournesol.app/index.php/Tournesol_Association).
+- To run tests locally, run `$ ./tests.sh` (does the same as Github CI)
 
 <details>
-  <summary>Click to expand</summary>
+  <summary>⚙ <b>Setting up your development environment on your computer</b> (click to expand)</summary>
+  
+### Tested platforms  
+**Docker setup on Ubuntu, Debian, Windows 10, Mac OS X, Mac OS X with the M1 chip**
+    
+For expert use without Docker, the setup was tested on WSL 2 from Windows 10 Preview (no Docker), Debian GNU/Linux 10 (buster) (no Docker), Ubuntu 20.04 and 18.04 (no Docker). Mac OS X setup without Docker is more complicated, as an X server is required for integration tests, as well as some linux-specific dependencies.
 
+### Building the Docker image
 First, clone this repo and `cd` to it.
 You will need [Docker](https://docs.docker.com/get-docker/) installed and configured.
```

Comment:
Can you say more about the "configured"? For me for example, I needed to increase the RAM (bump to 4G) and swap file size (bump to 2G) for everything to run smoothly